### DISCUSSION TITLE
Feature: Merchant's products feed

### DISCUSSION THREAD

**Discussion by PerezMarcelo** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#4] [Ideas]
It would be great to implement a feed where merchants can exhibit their products to the users. This way the buyer can select what they want and add it to their cart. We can also add a search bar where the users can look up exactly what they want, and see what's available. This can make the user experience simple and quick.

**Comment by CdoxHype** [comment]
Yes, and a small detail is to add recommendations to the search bar below, like Amazon and eBay do, so that the search is faster. For example, if you search for notebooks, that word will appear below, as well as other recommendations such as lab notebooks, normal notebooks, etc , and for this, the system will only search for things that are available, not things that have not existed on the page.

**Comment by david-baez-bravo** [comment]
A recommendations feed that predicts a users interest would be nice. 

This isn't necessary for the product but would be cool to code and implement. Some sort of graph algorithm or AI can be used for this. We would need to research how YouTube and Netflix deal with this.

**Comment by DiegoVazquez1** [comment]
A nice quality-of -life feature that could be added for searching are tabs relating to past items. For example, if the buyer has a chair in his cart or wishlist, having a tab of "furniture" would be cool. More of a wide search relating to past items or items in the wishlist. This feature is implemented in Spotify and Pinterest and makes browsing feel more smooth

**Comment by gcal77** [comment]
I feel like another feature that would make the buyer's feed more organized and easier to navigate would be to add categories or tabs to their homepage. These categories being for overall college subjects (Arts, Physics, P.E., etc.) and other domestic products (like food, furniture, pharmaceuticals, or other miscellaneous products). Sites like Amazon, eBay and Etsy have these features in their respective homepages as well, and Mercado Colegial Online would greatly benefit from having this implemented.
